apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java

```diff
@@ -184,7 +184,7 @@ private static Coverage transformPartB(MetricRegistry metricRegistry, Beneficiar
         coverage, CcwCodebookVariable.MS_CD, beneficiary.getMedicareEnrollmentStatusCode());
 
     addCoverageCodeExtension(
-        coverage, CcwCodebookVariable.B_TRM_CD, beneficiary.getPartATerminationCode());
+        coverage, CcwCodebookVariable.B_TRM_CD, beneficiary.getPartBTerminationCode());
```

Comment:
We are missing a unit test and a changelog entry.

Reply:
Can we change the partATerminationCode in the test data so that it is different than partBTerminationCode? That should make the test fail with the existing code and your change here will cause that test to pass. It should also necessitate changing the tests that validate partATerminationCode.

For the api changelog I'm not sure that I agree. The only reason why the JSON didn't change is because our test data has the same values in these fields. That is not necessarily true for real data (I assume, otherwise they would be the same field) so consumers of the API will potentially see different, correct data showing up after this change. fyi @bfausett77